packages/webapp/pages/[userId]/index.tsx

```diff
@@ -57,6 +60,7 @@ const ProfilePage = ({
   return (
     <div className="flex flex-col gap-6 px-4 py-6 tablet:px-6">
       <Readme user={user} />
+      {isStreaksEnabled && <ReadingStreaksWidget />}
```

Comment:
You'll only the the streaks widget on someone else's profile, if you have it enabled.